modules/assim.sequential/R/Analysis_sda_block.R

```diff
@@ -16,7 +16,7 @@
 ##' @description This function provides the block-based MCMC sampling approach.
 ##' 
 ##' @return It returns the `build.block.xy` object and the analysis results.
-##' @importFrom dplyr %>%
+##' @importFrom dplyr %>% purrr
```

Comment:
This doesn't look like the correct solution. First, don't do importFrom on any function that you can actually apply a namespace to in practice. Second, purrr isn't a function within dplyr so there's nothing to import.